### PR TITLE
Adding godocs for the 4 functions listed:

### PR DESCRIPTION
**Pull request by fireynis**
SqlTeamStore.UpdateLastTeamIconUpdate
SqlTeamStore.UpdateMembersRole
SqlTeamStore.UserBelongsToTeams
SqlTeamStore.GetTeamMembersForExport

#### Summary

Added in godoc for the four functions listed above

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/14002
JIRA TIcket: https://mattermost.atlassian.net/browse/MM-23004

### CODE REVIEW

**Review by jespino — APPROVED** [review]
Looks good to me. Thanks @fireynis! :confetti_ball:

**Comment by hanzei** [comment]
Awesome, looking forward to your next PR @fireynis :rocket:

File: store/sqlstore/team_store.go [hunk 1]
```diff
@@ -1268,6 +1268,9 @@ func (s SqlTeamStore) RemoveAllMembersByUser(userId string) error {
 	return nil
 }
 
+// UpdateLastTeamIconUpdate sets the last updated time for the icon based on the passed in teamId. The
+// LastTeamIconUpdate and UpdateAt fields are set to the passed in curTime. Returns nil on success and the error
+// otherwise
```
Comment (justinegeffen) [inline store/sqlstore/team_store.go:1273]:
```suggestion
// otherwise.
```

File: store/sqlstore/team_store.go [hunk 2]
```diff
@@ -1268,6 +1268,9 @@ func (s SqlTeamStore) RemoveAllMembersByUser(userId string) error {
 	return nil
 }
 
+// UpdateLastTeamIconUpdate sets the last updated time for the icon based on the passed in teamId. The
```
Comment (justinegeffen) [inline store/sqlstore/team_store.go:1271]:
```suggestion
// UpdateLastTeamIconUpdate sets the last updated time for the icon based on the parameter passed in teamId. The
```

File: store/sqlstore/team_store.go [hunk 3]
```diff
@@ -1268,6 +1268,9 @@ func (s SqlTeamStore) RemoveAllMembersByUser(userId string) error {
...
+// LastTeamIconUpdate and UpdateAt fields are set to the passed in curTime. Returns nil on success and the error
```
Comment (justinegeffen) [inline store/sqlstore/team_store.go:1272]:
```suggestion
// LastTeamIconUpdate and UpdateAt fields are set to the parameter passed in curTime. Returns nil on success and the error
```

File: store/sqlstore/team_store.go (hunk 1 above)
Comment (justinegeffen) [inline store/sqlstore/team_store.go:1273]:
```suggestion
// otherwise.
```

File: store/sqlstore/team_store.go [hunk 4]
```diff
@@ -1534,6 +1539,8 @@ func (s SqlTeamStore) UserBelongsToTeams(userId string, teamIds []string) (bool,
 	return c > 0, nil
 }
 
+// UpdateMembersRole updates all the members of teamID in the userIds string array to be admins and setting all other
```
Comment (justinegeffen) [inline store/sqlstore/team_store.go:1542]:
```suggestion
// UpdateMembersRole updates all the members of teamID in the userIds string array to be admins and sets all other
```

File: store/sqlstore/team_store.go [hunk 5]
```diff
@@ -1534,6 +1539,8 @@ func (s SqlTeamStore) UserBelongsToTeams(userId string, teamIds []string) (bool,
...
+// users to be not admin.
```
Comment (justinegeffen) [inline store/sqlstore/team_store.go:1543]:
```suggestion
// users as not being admin.
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, @fireynis, this is great. :)

**Review by fireynis — COMMENTED** [review]
Sounds good to me!

**Pull request merged**